diesel_tests/tests/migrations.rs

```diff
@@ -0,0 +1,65 @@
+use diesel::types::{Bool, BigInt};
+use diesel::{select, LoadDsl};
+use diesel::result::QueryResult;
+use diesel::connection::Connection;
+use diesel::expression::dsl::sql;
+use diesel::migrations::revert_migration;
+use schema::*;
+
+
+// FIXME: Remove the duplicates of this function once expression level attributes
+// are stable (I believe this is in 1.7)
```

Comment:
Unfortunately expression level attributes weren't stabilized in 1.7 :/ 
https://github.com/rust-lang/rust/blob/1.7.0/src/libsyntax/feature_gate.rs#L232